src/main/java/shrek/exception/InvalidCommandException.java

```diff
@@ -0,0 +1,17 @@
+package shrek.exception;
+
+public class InvalidCommandException extends RuntimeException {
+    public InvalidCommandException(String errorMessage, int errorCount) {
+        if (errorCount < 3) {
```

Comment:
Avoid using magic number such as `3`, consider extracting it out into a constant.